### ISSUE TITLE
False-positive when dot ending sentence is after URL

### ISSUE THREAD

**Issue by invidian** [tetafro/godot#12]
Following pieces of code triggers false positives it seems:
```go
const (
  // KubernetesCACN is a default CN for Kubernetes CA certificate, as recommended by
  // https://kubernetes.io/docs/setup/best-practices/certificates/.
  KubernetesCACN = "kubernetes-ca"

  // KubernetesFrontProxyCACN is a default CN for Kubernetes front proxy CA certificate,
  // as recommended by https://kubernetes.io/docs/setup/best-practices/certificates/.
  KubernetesFrontProxyCACN = "kubernetes-front-proxy-ca"

  // EtcdCACN is a default CN for etcd CA certificate, as recommended by
  // the https://kubernetes.io/docs/setup/best-practices/certificates/.
  EtcdCACN = "etcd-ca"
)
```

```console
$ golangci-lint run --disable-all --enable=godot --max-same-issues=0 --max-issues-per-linter=0 --build-tags integration,e2e --timeout 10m --exclude-use-default=false --sort-results ./...
pkg/pki/etcd.go:8:72: Comment should end in a period (godot)
        // EtcdCACN is a default CN for etcd CA certificate, as recommended by
                                                                              ^
pkg/pki/kubernetes.go:8:84: Comment should end in a period (godot)
        // KubernetesCACN is a default CN for Kubernetes CA certificate, as recommended by
                                                                                          ^
pkg/pki/kubernetes.go:12:88: Comment should end in a period (godot)
        // KubernetesFrontProxyCACN is a default CN for Kubernetes front proxy CA certificate,
                                                                                              ^
$ golangci-lint version
golangci-lint has version 1.33.0 built from b90551c on 2020-11-23T05:15:36Z
```

**Comment by richhx** [comment]
+1 also seeing this

**Comment by tetafro** [comment]
Fixed in 1.3.1.
FYI, I disabled check for comments with a URL at the end. I think forcing a period at the end of the URL could be confusing
```go
// Read more:
// http://example.com/?token=abc.123.
```
Not clear whether it's a part of the URL or not.
